src/features/offer/components/OfferBody/OfferBody.native.test.tsx

```diff
@@ -638,12 +648,69 @@ describe('<OfferBody />', () => {
       })
     })
   })
+
+  describe('same artist playlist with "wipSameArtistPlaylist" feature flag activated', () => {
+    beforeAll(() => {
+      mockSameArtistPlaylist = mockedAlgoliaOffersWithSameArtistResponse
+    })
+
+    beforeEach(() => {
+      useFeatureFlagSpy.mockReturnValueOnce(false).mockReturnValueOnce(true)
+    })
+
+    it('should display same artist list when offer has some', async () => {
+      renderOfferBody({
+        sameArtistPlaylist: mockSameArtistPlaylist,
+      })
+
+      await screen.findByText('Envoyer sur Instagram')
+
+      expect(screen.queryByTestId('sameArtistPlaylist')).toBeOnTheScreen()
+    })
+
+    it('should not display same artist lists when offer has not it', async () => {
+      renderOfferBody()
+
+      await screen.findByText('Envoyer sur Instagram')
+
+      expect(screen.queryByTestId('sameArtistPlaylist')).not.toBeOnTheScreen()
+    })
+  })
+
+  describe('same artist playlist with "wipSameArtistPlaylist" feature flag deactivated', () => {
+    beforeAll(() => {
+      mockSameArtistPlaylist = mockedAlgoliaOffersWithSameArtistResponse
+    })
+
+    beforeEach(() => {
+      useFeatureFlagSpy.mockReturnValueOnce(false).mockReturnValueOnce(false)
+    })
+
+    it('should not display same artist list when offer has some', async () => {
+      renderOfferBody({
+        sameArtistPlaylist: mockSameArtistPlaylist,
+      })
+
+      await screen.findByText('Envoyer sur Instagram')
+
+      expect(screen.queryByTestId('sameArtistPlaylist')).not.toBeOnTheScreen()
+    })
+
+    it('should not display same artist lists when offer has not it', async () => {
```

Comment:
`should not display same artist lists when offer has not it` (tu as un s en trop à list)